server/app/controllers/obs.py

```diff
@@ -15,13 +18,81 @@ def obs(obs_id: ObservationId = None, limit_and_offset = None):
     with repository.transaction():
         observation = repository.read_observation(obs_id)
 
+        orbit = None
+
         if observation is None:
             abort(404, "Observation not found")
-    
+
+        orbit = observation
+        if observation['tle'] is not None:
+            orbit = parse_tle(observation['tle'][0], observation['tle'][1])
```

Comment:
If ```obervation``` always has two lines then you can use
```parse_tle(*observation['tle'])``` little more pythonic, but now it is good too.

Reply:
Nice trick. Updated as suggested.